Partially resolve #655 Provide EntityTFPenguin with its own loot table.

noobanidus opened the pull request:
Partially resolves #655 by providing a default loot table (duplicated from the bird loot table) for EntityTFPenguin.

noobanidus committed:
Provide EntityTFPenguin with its own loot table.

Currently penguins use the EntityTFBird loot table by default, meaning
that adding additional drops to them in a simple manner can only be done
either by also adding those drops to tiny birds, or through ASM or
reflection.

Tamaized commented on src/main/java/twilightforest/entity/passive/EntityTFPenguin.java:
```diff
@@ -20,6 +20,8 @@
 
 public class EntityTFPenguin extends EntityTFBird {
 
+	public static final ResourceLocation LOOT_TABLE = new ResourceLocation(TwilightForestMod.ID, "entities/penguin");
```
Just creating a field wont work, you have to override `getLootTable`

noobanidus replied:
My bad, I've been working with Python for the past few days and didn't actually test modifying the new loot table. I forgot Java superclass functions don't care if you redefine a value in a subclass... whoops!

noobanidus committed:
Actually include getLootTable. Oops!

Turns out working with Python for a few days destroys my flimsy
understanding of subclassing and fields.

noobanidus commented:
I've included the requested and required change.
